doc/source/ray-air/doc_code/checkpoint_mlflow.py

```diff
@@ -0,0 +1,28 @@
+# flake8: noqa
+# isort: skip_file
+
+# __mlflow_checkpoint_start__
+from ray.air.checkpoint import Checkpoint
+from sklearn.ensemble import RandomForestClassifier
+import mlflow.sklearn
+
+# Create an sklearn classifier
+clf = RandomForestClassifier(max_depth=7, random_state=0)
+# ... e.g. train model with clf.fit()
+# Save model using MLflow
+mlflow.sklearn.save_model(clf, "model_directory")
+
+# Create checkpoint object from path
+checkpoint = Checkpoint.from_directory("model_directory")
+
+# Write it to some other directory
+checkpoint.to_directory("other_directory")
+# You can also use `checkpoint.to_uri/from_uri` to
```

Comment:
I'm a bit surprised that `.to_directory(dir)` is not just `.to_uri("file://" + dir)`. Why have two methods?